lib/exception_notifier.rb

```diff
@@ -96,10 +104,19 @@ def notifiers
     #   ExceptionNotifier.ignore_if do |exception, options|
     #     not Rails.env.production?
     #   end
+    #
+    # It also accepts a condition that is specific to each individual notifier.
+    #   ExceptionNotifier.ignore_if(notifier: :email) do |exception, options|
+    #     not Rails.env.production?
+    #   end
     def ignore_if(&block)
       @@ignores << block
     end
 
+    def ignore_notifier_if(notifier, &block)
```

Comment:
Here I put a separate method specifically for `ignore_notify_if` option based on the prior [suggestion](https://github.com/smartinez87/exception_notification/pull/419#discussion_r363719015).